Merge bed status batch update

fredkingham opened the pull request:
Handle batch bed status creation with merged MRNs

Commit:
Makes the bed status load work with merged MRNs

Changes the bed status load to use get_patients_from_mrns which also
handles MRNs with empty strings.

It also moves the creation of the patients outside the transaction block
so we don't send creation instructions for a patient id that is
subsequently rolled back.

Commit:
Merge branch 'prepayment' into merge-bed-status-batch-update

Commit:
Merge branch 'cerner-merge-branch' into merge-bed-status-batch-update

Commit:
Renaming get_patients_from_mrns to find_patients_from_mrns

Commit:
Adds another unit test to the batch bed status loader

davidmiller commented on plugins/admissions/loader.py:
```diff
@@ -354,29 +355,28 @@ def load_bed_status():
         for bed_data in status:
-            bed_status = BedStatus()
+            patient = mrn_to_patient.get(bed_data["Local_Patient_Identifier"])
+            if not patient:
```
I think what `if not patient` really means is `if bed_data["Local_Patient_Identifier"] == None || bed_data["Local_Patient_Identifier"] == '' || bed_data["Local_Patient_Identifier"].lstrip('0') == '' ` but done implicitly by another function in another file that throws away data it doesn't like? 

I predict that won't be immediately obvious in 6months - as a minimum we should explain how it happens

Commit:
Adds a comment to plugins.admissions.loader

Adds a comment to not patient to make it clear why a patient would not
be available within the mrn_to_patient mapping.
